dataplane/standalone/saiserver/routing.go

```diff
@@ -189,3 +245,29 @@ func newRouterInterface(mgr *attrmgr.AttrMgr, dataplane routingDataplaneAPI, s *
 	saipb.RegisterRouterInterfaceServer(s, r)
 	return r
 }
+
+// CreateRouterInterfaces creates a new router interface.
+func (ri *routerInterface) CreateRouterInterface(ctx context.Context, req *saipb.CreateRouterInterfaceRequest) (*saipb.CreateRouterInterfaceResponse, error) {
+	id := ri.mgr.NextID()
+	iReq := &dpb.AddInterfaceRequest{
+		Id:      fmt.Sprint(id),
+		VrfId:   uint32(req.GetVirtualRouterId()),
+		Mtu:     uint64(req.GetMtu()),
+		PortIds: []string{fmt.Sprint(req.GetPortId())},
+		Mac:     req.GetSrcMacAddress(),
+	}
+	switch req.GetType() {
+	case saipb.RouterInterfaceType_ROUTER_INTERFACE_TYPE_PORT:
+		iReq.Type = dpb.InterfaceType_INTERFACE_TYPE_PORT
+
+	case saipb.RouterInterfaceType_ROUTER_INTERFACE_TYPE_LOOPBACK: // TODO: Support loopback interfaces
```

Comment:
should this log a warning or something to indicate that its not yet fully supported? or maybe even return an error

Reply:
done